cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  const email = 'mlemeshev@i.ua';
```

Comment:
It's a good practice to keep test data such as usernames and passwords in a separate config or data file. This makes it easier to change the data and also keeps it secure.